index.js

```diff
@@ -21,33 +18,25 @@ fs.mkdir(pathName, { recursive: true }, (error) => {
 
 fs.readFile(file, 'utf8', (err, data) => {
     if (err) {
-        return console.log(err)
+        throw err;
     }
 
     // the data is split into a new array called clippings
     const clippings = data.split('==========');
 
     // declare an empty array that will hold the books & highlights
-    let books = [];
-
-    // searchBooks searches in 'books' array & returns index of book if it exists
-    function searchBooks(books, title) {
-        for(let i = 0; i < books.length; i++) {
-            if (books[i].title === title) {
-                bookIndex = i;
-                return;
-            }
-        }
-        bookIndex = -1;
-        return bookIndex;
-    }
+    const books = {};
 
     for(let i = 0; i < clippings.length; i++) {
 
         // split each clipping in clippings into component parts whenever there's a new line
-        let clipping = clippings[i].split('\r\n');
+        let clipping = clippings[i].split('\n');
```

Comment:
seems like there are multiple types of whitespace characters used here, and it seems like splitting on any one of them would work. so maybe this line should be changed to `let clipping = clippings[i].split(/[\r\n]+/);`